app/services/arcgis_api/geocoder.rb

```diff
@@ -1,8 +1,17 @@
 module ArcgisApi
   class Geocoder
     Suggestion = Struct.new(:text, :magic_key, keyword_init: true)
+    AddressCandidate = Struct.new(
+      :address, :location, :street_address, :city, :state, :zip_code,
+      keyword_init: true
+    )
+    COMMON_DEFAULTS = {
+      f: 'json',
+      countryCode: 'USA',
+      category: 'address',
+    }
 
-    # Makes HTTP request to get potential address matches
+    # Makes HTTP request to quickly find potential address matches
```

Comment:
Just a non-blocking suggestion but I think it'd be useful to call out the link between #suggest and #find_address_candidates from both methods

```suggestion
    # Makes an HTTP request to quickly find potential address matches. Each match that is found
    # will include an associated magic_key value which can later be used to get more details about
    # the address using the #find_address_candidates method
```